### PR TITLE
Release Java worker 2.0.0

### PR DESCRIPTION
**Pull request by kaibocai**
https://github.com/Azure/azure-functions-java-worker/releases/tag/2.0.0

### CODE REVIEW

**Review by soninaren — APPROVED** [review]

File: NuGet.Config [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <add key="azure_app_service_staging" value="https://www.myget.org/F/azure-appservice-staging/api/v3/index.json" />
     <add key="Microsoft.Azure.Functions.PowerShellWorker" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/Microsoft.Azure.Functions.PowerShellWorker/nuget/v3/index.json" />
     <add key="AzureFunctions" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctions/nuget/v3/index.json" />
-    <add key="AzureFunctionsPreRelease" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsPreRelease/nuget/v3/index.json" />
+    <add key="AzureFunctionsRelease" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsRelease/nuget/v3/index.json" />
+	<add key="AzureFunctionsPreRelease" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsPreRelease/nuget/v3/index.json" />
```
Comment (soninaren) [inline NuGet.Config:10]: nit: extra space here.

**Review by soninaren — APPROVED** [review]

File: NuGet.Config (hunk 1 above)
Comment (kaibocai) [reply NuGet.Config]: Updated please check. Thanks.

**Pull request merged**